lib/experimental/Widgets/Content/IconText/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { Icon } from "@/components/Utilities/Icon"
+import BriefcaseIcon from "@/icons/Briefcase"
+import HomeIcon from "@/icons/Home"
+import OfficeIcon from "@/icons/Office"
+import { forwardRef } from "react"
+
+const iconsMap = {
+  office: OfficeIcon,
+  briefcase: BriefcaseIcon,
+  home: HomeIcon,
+}
+
+interface IconTextProps {
+  icon: keyof typeof iconsMap
+  texts: string[]
```

Comment:
I'm not sure about this prop change, why not joining the text in our frontend and passing it as a `text`  string prop as we were doing before?

Reply:
Because now there's a small dot between the values, I need to insert it between them.